danggunMarket/src/main/java/com/example/danggunmarket/member/MemberController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.danggunmarket.member;
+
+import com.example.danggunmarket.member.dto.JoinRequest;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+public class MemberController {
+    private final MemberService memberService;
+
+    @PostMapping("/v1/signup")
```

Comment:
```suggestion
    @PostMapping("/v1/members")
```
요렇게 의미표현이 충분해 보이기도 하네요.